test/contrib/flag/test_flag.py

```diff
@@ -107,3 +108,58 @@ def test_zero_grad(model: torch.nn.Module, optimizer: torch.nn.Module,
         for p in group['params']:
             assert torch.count_nonzero(
                 p.grad) == 0, "After running FLAG, the gradients should be 0!"
+
+
+class FLAGLossHistoryCallback(FLAGCallback):
```

Comment:
not a blocker, but maybe this should be moved to `flag_callback.py` so that someone looking at the callback class has an example of the types of callbacks they can create.

Reply:
I was thinking about that too, I can see an argument for `FLAGLossHistoryCallback` being moved to that file, but not the counter one — that feels more like a test template rather than something that should be included in the actual codebase

Reply:
Agreed 100%